### DISCUSSION TITLE
⚡️SuperEditor in the wild! Are you using SE in your live app? Comment here!

### DISCUSSION THREAD

**Discussion by daverad** [Flutter-Bounty-Hunters/super_editor#1032] [Q&A]
**Who else is using Super Editor in production on iOS, Android, Mac and/or Web?**

Would be great to see who else in the community is using SE and in what capacity.

👏 An opportunity to celebrate each others work!

🤓 And helpful from a QA perspective to understand if some issues are specific to our individual apps or global issues.

**Comment by daverad** [comment]
**App Name:** Reflection.app

**App Description:** Your guided journal for wellness and growth.

**Platforms:** iOS, Android, MacOS, Web

**Link to app:** https://www.reflection.app/#download

**Comment by matthew-carroll** [comment]
Would other funding companies like to describe their uses? @miguelcmedeiros @Jethro87 @jmatth @hillelcoren @venkatd

**Comment by hillelcoren** [comment]
App Name: Invoice Ninja

App Description: Invoicing Software for Small Businesses

Platforms: iOS, Android, MacOS, Windows, Linux and Web

Link to app: https://demo.invoiceninja.com

We're using the Super Editor to enable people to create/preview custom email templates with markdown.

**Comment by Jethro87** [comment]
**App Name**: Clearful Journal

**App Description**: The Best Journal for Personal Growth

**Platforms**: iOS and Android. MacOS and Web coming soon.

**Link to app**: https://clearful.com/

The core journal writing/editing experience, with in-line formatting and block-level nodes, is built with Super Editor.

**Comment by miguelcmedeiros** [comment]
**App Name:** Superlist

**App Description:** Productivity app made for teams, designed for people

**Platforms:** MacOS, iOS and Android. Web and Windows coming soon.

**Link to app:** https://www.superlist.com/
Currently we are still in a closed beta, but we plan to invite more beta users soon 😉 

We use both SuperEditor (editing lists which include paragraphs, headers, attachments, etc. and of course, tasks) and SuperTextField (all text fields) across the app.
